src/lib/docs-gen/src/helpers/translation-extract-helper.ts

```diff
@@ -39,26 +41,32 @@ export class TranslationExtractHelper {
     }
   }
   private generateTitle(title: string, propertyName: string) {
-    if (title && title.trim().length > 0) return title;
+    if (title && title.trim().length > 0) {
+      return title;
+    }
 
     const newTitle = `${propertyName}`
-      .replace(new RegExp(/[^\w]/, 'g'), ' ') //Remove non-word characters such as '-'
+      .replace(new RegExp(/[^\w]/, 'g'), ' ') // Remove non-word characters such as '-'
       .replace(
-        new RegExp(/(\w)(\w*)/, 'g'), //Grab first character and remainder of the word so we can Pascale Case it
+        new RegExp(/(\w)(\w*)/, 'g'), // Grab first character and remainder of the word so we can Pascale Case it
         ($1, $2, $3) => {
           return `${$2.toUpperCase() + $3.toLowerCase()}`;
         },
       );
     return newTitle;
   }
   private translate(key: string, selectedLanguage: string) {
-    if (selectedLanguage === 'fr') return frenchDict.fr[key];
-    else return englishDict.en[key];
+    if (selectedLanguage === 'fr') {
```

Comment:
The else here is probably extraneous
`    if (selectedLanguage === 'fr') {
      return frenchDict.fr[key];
    }
    return englishDict.en[key];
      `